src/main/antlr4/imports/mysql_idents.g4

```diff
@@ -4,6 +4,7 @@ import mysql_literal_tokens;
 
 db_name: name;
 table_name: (db_name '.' name)
+            | '.' name // *shakes head in sadness*
```

Comment:
the sins of our father-engineers will snowball into massive technical debts one day.  If they haven't already.